### ISSUE TITLE
Error: "Update dependency jest to v26"

### ISSUE THREAD

**Issue by bryndyment** [stefanoeb/jest-action#7]
Hi!

```
FAIL ./mock.spec.js
  ● Test suite failed to run

    ReferenceError: it is not defined

      1 | const capitalize = string => (string ? `${string[0].toUpperCase()}${string.slice(1)}` : '')
      2 | 
    > 3 | it('capitalizes', () => expect(capitalize('hoo')).toEqual('Hoo'))
        | ^
      4 | 

      at Object.<anonymous> (mock.spec.js:3:1)

Test Suites: 1 failed, 1 total
Tests:       0 total
Snapshots:   0 total
Time:        1.105 s
Ran all test suites.
```

↑ Getting this error when `renovate` attempts to update `jest` to `v26`.

Help?

**Comment by stefanoeb** [comment]
Does jest succeeds when being executed locally?

**Comment by bryndyment** [comment]
Yes:

![image](https://user-images.githubusercontent.com/7953560/82133868-cb4c1c80-982b-11ea-96dd-dcd47c3ea3e4.png)

Before running `jest`, I did this:

![image](https://user-images.githubusercontent.com/7953560/82133887-fa628e00-982b-11ea-9097-0d1e17fae411.png)

Thanks!

**Comment by axe312ger** [comment]
I can confirm this issue. Jest v26 does not work, we would need to update the docker container by one fix version

```
error jest@26.0.1: The engine "node" is incompatible with this module. Expected version ">= 10.14.2". Got "10.14.1"
```


My workaround: Lock version of jest to v25 till this action is updated

**Comment by axe312ger** [comment]
@stefanoeb just seen you already updated the docker container. So a small release to npm could fix this already :)

**Comment by BraveLilToaster** [comment]
@stefanoeb is there something else blocking a new release. I would happy to help if I am able.

**Comment by stefanoeb** [comment]
Published [1.0.4](https://github.com/stefanoeb/jest-action/compare/1.0.3...1.0.4)
